src/factory/src/state.rs

```diff
@@ -49,11 +52,35 @@ impl Default for State {
     }
 }
 
-pub fn get_token_bytecode() -> &'static [u8] {
-    include_bytes!("token.wasm")
+pub fn get_token_bytecode() -> Vec<u8> {
+    State::get().borrow().token_wasm.clone().unwrap_or_default()
```

Comment:
Or you could actually panic here if there's no code to use. So that any method that tries to use the code when there's none would not be able to do so.